addons/print/wizard/print_order_sendnow_wizard.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class print_order_sendnow_wizard(models.TransientModel):
+
+    _name = 'print.order.sendnow.wizard'
+    _rec_name = 'create_date'
+
+    # --------------------------------------------------
+    # MODEL FIELDS
+    # --------------------------------------------------
+    print_order_sendnow_line_wizard_ids = fields.One2many('print.order.sendnow.line.wizard', 'print_order_sendnow_wizard_id', string='Lines')
+
+    # --------------------------------------------------
+    # METHODS
+    # --------------------------------------------------
+    @api.model
+    def default_get(self, fields):
+        """ create the lines on the wizard """
+        res = super(print_order_sendnow_wizard, self).default_get(fields)
+        active_ids = self._context.get('active_ids', [])
+        if active_ids:
+            # create order lines
+            lines = []
+            for rec in self.env['print.order'].browse(active_ids):
```

Comment:
for rec in self.env[].search[('id', in, 'active_ids'), ('state', '!=', 'sent')] ? This allows to browse only effectively necessary records (and remove one level of indentation).